### DISCUSSION TITLE
Customize Trade Output

### DISCUSSION THREAD

**Discussion by sprocketx** [aussig/BGS-Tally#157] [Ideas]
Would it be possible to add an option to show either each trade profit demand, a summary of total trade profits or both.   Would help make a tighter output when its not necessary to post those details depending on the discord group.

**Comment by aussig** [comment]
Currently we've got:

`TrdBuy 🅻:123k 🅼:234k 🅷:345k TrdProfit 🆉:123k 🅻:234k 🅼:345k 🅷:456k`

But you would like a simpler option, combining the values:

`TrdBuy 702k TrdProfit 1.15M`

This is very similar to how it used to be before BGST started tracking each demand level separately. The only issue I can see is further expansion and complexity of the 'settings' panel.  This is likely to expand further when I finally get around to implementing #70.

**Comment by sprocketx** [comment]
Yeah, that's what I'm asking. The bgs group I'm working with doesn't track per demand value.  I understand if this wouldn't be a very popular ask or a pretty niche situation.  Might not be used beyond me :)  Thanks for considering it!

**Comment by aussig** [comment]
This is implemented in [3.5.0-a1](https://github.com/aussig/BGS-Tally/releases/tag/3.5.0-a1)
